eth/beacon/helpers.py

```diff
@@ -268,11 +267,14 @@ def _get_shards_and_committees_for_shard_indices(
 
 
 @to_tuple
-def get_new_shuffling(seed: Hash32,
+def get_new_shuffling(*,
```

Comment:
:heart: 

I just deleted a comment suggesting that you adopt this exact pattern!

One thing to think about.  Invoking functions using keyword arguments has a runtime cost.  I suspect that for this function, that runtime cost is going to be trivial compared to the execution of the body of the functions, but for any function that needs to be highly performant, this should be taken into account.

Reply:
*If* we move it into StateMachine, we can reduce some parameters by using  `self.config.CYCLE_LENGTH`, `self.config.min_committee_size`, `self.config.shard_count`.